README.md

```diff
@@ -27,84 +27,91 @@ cargo build --release
 
 ## Run
 
+### Testnets
+
+Centrifuge has multiple testnets online.
+
+1. Fulvous is an ephemeral testnet that is tracking `master`.
+2. Flint is for breaking changes and testing the integration with other parts of the Centrifuge ecosystem. Think of Flint as a way to test previews, alpha releases.
+3. Amber is for audits and testing of the stability of release candidates. Think of Amber as a way to test beta releases.
+
+#### 1. Fulvous
+
+To run a node:
+
+```bash
+./target/release/centrifuge-chain \
+  --chain=fulvous \
+  --name "My node name" \
+  --bootnodes /ip4/35.246.140.178/tcp/30333/p2p/QmRg2bEPTHCt8u3a1LeZA8dJTd8mgMccsAcoHXTjQUpcZj \
+  --bootnodes /ip4/35.198.166.26/tcp/30333/p2p/QmNpeu3bJhESzriWMLRcxRgSCYDGQ6GdBHnJAf8bJexAd5
+```
+
+#### 2. Flint
+
+To run a node:
+
+```bash
+./target/release/centrifuge-chain \
+  --chain=flint \
```

Comment:
Do we need to ensure here the usage of the specific genesis version, instead of latest. 
`--chain=flint-cc2-spec.json`

Reply:
No, `--chain=flint` will resolve to the latest file, since that's loaded in `chain_spec.rs`